lms/envs/minimal.yml

```diff
@@ -0,0 +1,22 @@
+# WARNING: Experimental
+#
+# This is the minimal settings you need to set to be able to get django to
+# load when using the production.py settings files. It's useful to point
+# LMS_CFG and CMS_CFG to this file to be able to run various paver commands
+# without needing a full docker setup.
+---
+
+SECRET_KEY: aseuothsaeotuhaseotisaotenihsaoetih
```

Comment:
This is clearly a Dvorak keymash. I think we should keep that kind of pretension out of the codebase.